### DISCUSSION TITLE
Cron monitoring on developing stage

### DISCUSSION THREAD

**Discussion by lloricode** [getsentry/sentry-laravel#695] [Q&A] [answered]
I have a scheduled command

```php

        $schedule->command(SomeCommand::class)
            ->sentryMonitor('point-of-sale-cron-monitoring')
```

When I try to run ` php artisan schedule:list` on my local, it's working fine, but when `SENTRY_LARAVEL_DSN=` in .env is not set it will throw an exception:

```
Method Illuminate\Console\Scheduling\Event::sentryMonitor does not exist.
```

I hope this allow to just ignore when there is no setup for sentry

**Comment by stayallive** [comment, marked as answer]
This is definitly a bug and not intended, I've created an issue to track this: #696.
